apps/rotm/views/add-image.html

```diff
@@ -0,0 +1,16 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#image-preview}}
+      <div class="image-wrapper">
+        <img src="{{image-preview}}" alt="{{image-name}}" width="200" class="image-preview" />
```

Comment:
I'm not sure setting alt text is useful in this context since we can't meaningfully describe the image content because we don't know what's in it.

Reply:
A screen reader will skip an image entirely when the the alt tag is not included or included but empty. Would you consider it useful to provide alt text that says something like "This is a preview of the image you uploaded"?

Reply:
I'm not certain that adds any particular value, especially since we also provide the image name in plain text immediately below.

As I understand it, the current screenreader flow would be "Is this correct? _[name of image file]_ Yes/No", which is about as good a flow as we could hope for I think. I certainly don't think that adding extra words in the middle would make it easier to use.

(this is all slightly disregarding the fact that it's fairly unlikely that visually impaired users would be uploading images anyway)